table/block_based/partitioned_index_reader.cc

```diff
@@ -230,9 +229,10 @@ void PartitionIndexReader::EraseFromCacheBeforeDestruction(
   if (uncache_aggressiveness > 0) {
     CachableEntry<Block> top_level_block;
 
-    GetOrReadIndexBlock(/*no_io=*/true, /*get_context=*/nullptr,
-                        /*lookup_context=*/nullptr, &top_level_block,
-                        ReadOptions{})
+    ReadOptions ro_no_io;
```

Comment:
Is this intentional that we don't pass down read options here?

Reply:
We aren't in the context of a read operation (no ReadOptions passed in), but we are in a context where we are very much forbidden from reading from storage, because that would destroy the purpose of pro-actively erasing obsolete cache entries.